### DISCUSSION TITLE
Error in vps

### DISCUSSION THREAD

**Discussion by danielgap** [omkarcloud/google-maps-scraper#57] [Q&A]
When I run the script on a windows server VPS I constantly get the following error: "[14968:12952:1024/205250.057:ERROR:gles2_cmd_decoder_passthrough.cc(947)] ContextResult::kFatalFailure: fail_if_major_perf_caveat + swiftshader".
Is it possible to do something in the selenium configuration so that it doesn't come out?

**Comment by Chetan11-dev** [comment]
It is recommended to install Docker on Windows. After installation, run the following commands to scrape places:

```
docker-compose build && docker-compose up
```

**Comment by Chetan11-dev** [comment]
If you wish to run the application on a server, consider using Container Optimized OS on Google Cloud's Compute Engine that comes with Docker. More details can be found at [Google's documentation](https://cloud.google.com/container-optimized-os/docs). 

After setting it up, you can upload your repository and execute the following command to initiate the scraping process:

```
docker-compose build && docker-compose up
```
